src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ResourcesPropertyPage.xaml

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
+<Rule Name="ResourcesPropertyPage"
+      Description="Resources"
+      DisplayName="Resources"
+      PageTemplate="generic"
+      Order="600"
+      xmlns="http://schemas.microsoft.com/build/2009/properties">
+
+  <Rule.DataSource>
+    <DataSource Persistence="ProjectFileWithInterception"
+                SourceOfDefaultValue="AfterContext"
+                HasConfigurationCondition="False" />
+  </Rule.DataSource>
+
+  <StringProperty Name="ResourcesPagePlaceholderDescription"
+                  DisplayName="Ignored"
```

Comment:
Is this a special value to make it not use DisplayName? Interesting that something like empty string or excluding the attribute entirely wouldn't do the same thing.

Reply:
No it's not special. Just capturing the fact that it's unused in any way. Without this `Name` is used in place of `DisplayName`. Omitting `DisplayName` altogether would avoid translating this string, but we have unit tests that verify all visible properties have a `DisplayName` value.